cmd/payload-testing-prow-plugin/server.go

```diff
@@ -154,6 +161,19 @@ func jobsFromComment(comment string) []config.Job {
 		})
 
 	}
+	for _, match := range ocpPayloadJobTestsWithPRsPattern.FindAllStringSubmatch(comment, -1) {
```

Comment:
Should we set an upper bound on how many PRs can be included?

Reply:
hmm, that is an interesting thought. There is a 6 hour timeout on the job itself, so if you include too many than it will likely end up failing. As far as what "too many" is, I am unsure. I tried it with 5 and it was fine. I don't think we need to impose an arbitrary limit here. Let's wait and see how it is used and if we notice it running into problems we can adjust it. I think the majority of the time this will be run with 2 - 3 prs.